smtp/lib.c

```diff
@@ -265,6 +275,7 @@ luaT_smtpc_cleanup(lua_State *L)
 
 static const struct luaL_Reg Module[] = {
 	{"new", luaT_smtpc_new},
+	{"__CURL_VERSION", luaT_smtpc_version},
```

Comment:
Can we call it in `luaopen_smtp_lib()` and fill the field using `lua_setfield()`?

The uppercased functoin name looks quite unusual.

Or lowercase it.

Reply:
I see, it is called in `init.lua` and stored as a field. Okay. So, up to you.